mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/IndexableMessageTest.java

```diff
@@ -482,5 +483,97 @@ public void messageShouldBeIndexedEvenIfTikaParserThrowsAnError() throws Excepti
         // Then
         assertThat(indexableMessage.getText()).contains("subject should be parsed");
     }
+    
+    @Test
+    public void shouldHandleCorrectlyMessageIdWhichSerializeExceptionThrowAException() throws Exception {
```

Comment:
Ok I make a mistake but this wasn't Which/With it was
should handle correctly message id that serialize method throw an exception
-> shouldHandleCorrectlyMessageIdThatSerializeMethodThrowAnException